### ISSUE TITLE
Am I missing something here?

### ISSUE THREAD

**Issue by mysteriousHerb** [Extraltodeus/LoadLoraWithTags#3]
![image](https://github.com/Extraltodeus/LoadLoraWithTags/assets/13511170/5f67d8c4-46a7-469b-b25b-3fa10e4f237d)
How do you really use it? In the attached setting, it didnt really append the key words?

**Comment by opensourcefan** [comment]
Yes... I have the string output going into a context node as well as a "show text" node for fun. From there I have a link from the context conditioning to a combine conditioning node which also has the normal positive prompt connected.

Maybe there's other ways to do it but this works perfect for me. The only issue is you'll obviously get all the triggers into the conditioning, not a specific one.

Not home so can't provide a screenshot sorry.

**Comment by opensourcefan** [comment]
Hope this helps, I'm remote with my server so not an elegant SS.

![Screenshot_2023-10-07-14-55-58-31_984e1414ae90666a90f12ff17ec14a7f.jpg](https://github.com/Extraltodeus/LoadLoraWithTags/assets/10550038/eb27c202-4129-48e3-a5e1-7071e88fb80a)

**Comment by Extraltodeus** [comment]
The keywords goe through the optional string input and out from the string output

**Comment by asidsx** [comment]
> The keywords go through the optional string input and out from the string output

Can you show example?

**Comment by TomLucidor** [comment]
@Extraltodeus but how do you select the specific subtype of keywords (for character packs and utility packs)?

**Comment by oliverban** [comment]
Many good questions but few good answers! :(

**Comment by Extraltodeus** [comment]
> @Extraltodeus but how do you select the specific subtype of keywords (for character packs and utility packs)?

I've never used such. The main goal is really to either have the entirety of the keywords joined to the prompt or to print them in the terminal and use/select them manually.

That's how I use the node

![image](https://github.com/Extraltodeus/LoadLoraWithTags/assets/15731540/618d4ced-9849-462f-a4aa-3a58ae4e5fd3)

**Comment by ewebgh33** [comment]
If it prints to terminal could it not also print to an integrated text box in the node itself? Rather than having to go out to a string or show text. Just parse on lora load and print triggers to a text box at the bottom of the node?
Have not coded nodes myself (yet) so apologies if my understanding is faulty.
